initrd/bin/gui-init

```diff
@@ -214,13 +227,13 @@ update_hotp()
       fi
     fi
     # Don't output HOTP codes to screen, so as to make replay attacks harder
-    hotp_verification check $HOTP
+    hotp_verification check "$HOTP"
     case "$?" in
       0 )
         HOTP="Success"
         BG_COLOR_MAIN_MENU=""
       ;;
-      4 )
+      4|7 )	# 4: code was incorrect, 7: code was not a valid HOTP code at all
```

Comment:
7 happens here if `HOTP` contained some "Unable to ...." error from unseal-hotp.  It doesn't mean that the token is not connected, it means we don't have a code, so I added it here.  Also quoted "$HOTP" above, otherwise the error was word-split and triggered an invalid-arguments error from hotp_verification.